OpenEdXMobile/res/drawable/view_selector.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <selector xmlns:android="http://schemas.android.com/apk/res/android">
-    <item android:drawable="@color/neutralDark" android:state_pressed="true" />
+    <item android:drawable="@color/neutralXLight" android:state_pressed="true" />
```

Comment:
Any reason for this change?

Reply:
The colours for the selected and pressed are the same in this [design](https://www.figma.com/file/Vaw3lfBXmOhTLKFnBedqE1/App-Navigation-Improvements?node-id=1474%3A13223&mode=dev).

Reply:
Plus the selection should be lighter than the borders. 🤷🏻